Assets/Scripts/Inputs/ControlPoint.cs

```diff
@@ -19,7 +19,8 @@ public class ControlPoint : MonoBehaviour
 
     private double? _chargeStartTime;
 
-    private bool _isPlanetLaunched;
+    [NonSerialized]
```

Comment:
what's the point of having NonSerialized attribute?

Reply:
In order to prevent changing the value in the inspector. By default, public properties can be edited in inspector.